csunplugged/tests/search/management/test_commands.py

```diff
@@ -0,0 +1,79 @@
+"""Module for the testing custom Django commands."""
+
+from tests.BaseTestWithDB import BaseTestWithDB
+from django.core import management
+from django.test import tag
+from tests.topics.TopicsTestDataGenerator import TopicsTestDataGenerator
+from tests.resources.ResourcesTestDataGenerator import ResourcesTestDataGenerator
+
+
+@tag("management")
+class ManagementCommandTest(BaseTestWithDB):
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.language = "en"
+        self.test_data = TopicsTestDataGenerator()
+
+    def test_rebuild_index_command_no_items(self):
+        management.call_command("rebuild_index", "--noinput")
```

Comment:
should these methods be asserting something? how does this magic work?

Reply:
Address via Slack. Is management called for building search index.